schema.sql

```diff
@@ -1,11 +1,14 @@
 /* Database schema to keep the structure of entire database. */
-
 CREATE TABLE animals (
-    id INT NOT NULL,
+	id INT NOT NULL,
 	name varchar(100) NOT NULL,
 	date_of_birth DATE NOT NULL,
 	escape_attempts INT NOT NULL,
 	neutered BOOLEAN NOT NULL,
 	weight_kg DECIMAL(5, 2) NOT NULL
 );
 
+alter table
+	animals
+add
+	species varchar(100) NOT NULL;
```

Comment:
- [ ] Hi @Victor-chiemerie, you did a good job so far but I can see you still need to understand the `psql` attributes or property. You are trying to add the new `column` **species** but you need to understand you are adding the column which will get data entered by the user, so by default `animals` table should alter without the `not null` property, so that it will allow the database to add the column without any error.

![image](https://github.com/Victor-chiemerie/Vet-Clinic-Database/assets/66476329/b4961f67-a782-4cd7-b934-bd41a0e75681)